front/pages/w/[wId]/index.tsx

```diff
@@ -17,17 +18,29 @@ export const getServerSideProps: GetServerSideProps = async (context) => {
     };
   }
 
-  if (owner.role === "user") {
+  // check if upsertable data sources using getDataSources
```

Comment:
Let's not change the default redirect. We still want to go to /u for users. If they have data sources and want to update them the builders/admin of their team will share the URL with them for now :+1: